ncm-profile: missing directory

If I install a new machine, I get this error from ncm-profile-13.3.0-1.noarch:
[ERROR] Error creating managed script list (/usr/lib/ncm/config/profile/managed_files)

The problem is that the directory "/usr/lib/ncm/config/profile" is not automatically created.

In the old ncm-profile-2.1.3-1.noarch, this directory was part of the RPM.

--- a/ncm-profile/pom.xml
+++ b/ncm-profile/pom.xml
@@ -44,4 +44,24 @@
     </contributor>
   </contributors>
 
+  <build>
+    <plugins>
+      <plugin>
+	<groupId>org.codehaus.mojo</groupId>
+	<artifactId>rpm-maven-plugin</artifactId>
+	<configuration>
+	  <mappings combine.children="append">
+	    <mapping>
+	      <directory>/usr/lib/ncm/config/profile/managed_files/</directory>
+	      <filemode>755</filemode>
+		<username>root</username>
+		<groupname>root</groupname>
+		<directoryIncluded>true</directoryIncluded>
+	      </mapping>
+	    </mappings>
+	</configuration>
+      </plugin>
+    </plugins>
+  </build>
+
 </project>

